metadata.json

```
@@ -1,6 +1,6 @@
 {
   "name": "puppet-icinga2",
-  "version": "5.0.2-rc0",
+  "version": "6.0.1-rc0",
```

If you want to do a 6.0.0 release, please update this to 6.0.0
```suggestion
  "version": "6.0.0",
```